src/button/button-group.scss

```diff
@@ -2,11 +2,18 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import '../style/index';
 
-@mixin iui-button-group {
+/// Mixin for button group or input+button combo group.
+/// @arg $orientation must be one of: horizontal, vertical
+/// Note: Vertical button-group should be used for icon buttons only.
+@mixin iui-button-group($orientation: horizontal) {
   display: inline-flex;
   align-items: center;
   isolation: isolate;
 
+  @if $orientation != 'horizontal' {
```

Comment:
Mainly because there is no type checking in Sass. I'm hardcoding `horizontal` as the default value, but the user of this mixin could pass anything. `@include iui-button-group(𝓿𝓮𝓻𝓽𝓲𝓬𝓪𝓵)`